index.js

```diff
@@ -78,20 +78,43 @@ const getSecurityVulnerabilitiesForBitbucketRepo = async (
   repo
 ) => {
   console.log(`Looking for vulnerabilities in Bitbucket ${repo}`)
+  let allVulnerabilities
+  
+  const javaScriptDependencies = await bitbucket.getJavaScriptDependenciesOfRepo(
+    bitbucketUsername,
+    bitbucketAppPassword,
+    repo
+  )
+  if (javaScriptDependencies === undefined) {
+    console.log('(No JavaScript dependencies found)')
+  } else {
+    const javaScriptVulnerabilities = await getSecurityVulnerabilitiesForJavaScriptDependencies(
```

Comment:
I admire your love of long names. I am not patient (nor accurate) enough to type them correctly!